database_scripts/mssql/006_User_Organisation_update.sql

```diff
@@ -0,0 +1,13 @@
+
+IF NOT EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = 'user_organisation' AND COLUMN_NAME = 'status')
+    BEGIN
+        ALTER TABLE user_organisation ADD [status] smallint DEFAULT 0 NOT NULL
```

Comment:
If I have read the rest of the PR correctly, then 0 as a default with no further action will cause some issue. It will mean any existing users in the system (which are approved by definition of using the system) will become unapproved. Either the default should change, or probably more sensibly, an update should be run straight after to update all existing users to approved.